### DISCUSSION TITLE
Swap between MIDI and IDE

### DISCUSSION THREAD

**Discussion by Marsupio** [FortySevenEffects/arduino_midi_library#302] [Q&A] [answered]
Probably i am missing something, but since the Arduino is recognized as a MIDI device, how you set it back to the default for IDE programming? Are you supposed to change something, or the IDE just keeps on seeing the device as before? (Last time i did it, I had to do bothersome stuff as flashing bootloader!)

**Comment by lathoub** [comment, marked as answer]
The Arduino is programmed through the USB interface, MIDI goes over the serial port - so there is no confusion over port usage. (Or are you using MIDI over USB?)

**Reply by Marsupio** [reply]
Excuse me, I meant MIDI over USB, I am afraid i misplaced my question

**Reply by lathoub** [reply]
Better ask it here: https://github.com/lathoub/Arduino-USBMIDI

**Reply by Marsupio** [reply]
Thanks. Please delete this thread

**Reply by lathoub** [reply]
>> Please delete this thread
Just mark it as answered
